doc/reference/modules/batch.xml

```diff
@@ -2,29 +2,31 @@
     <title>Batch processing</title>
     
     <para>
-        A naive approach to inserting 100 000 rows in the database using NHibernate might 
+        A naive approach to inserting 100 000 rows in the database using NHibernate might
         look like this:
     </para>
 
-<programlisting><![CDATA[ISession session = sessionFactory.OpenSession();
-ITransaction tx = session.BeginTransaction();
-for ( int i=0; i<100000; i++ ) {
-    Customer customer = new Customer(.....);
-    session.Save(customer);
-}
-tx.Commit();
-session.Close();]]></programlisting>
+<programlisting><![CDATA[using (ISession session = sessionFactory.OpenSession())
+using (ITransaction tx = session.BeginTransaction())
+{
+    for (int i = 0; i < 100000; i++)
+    {
+        Customer customer = new Customer(.....);
+        session.Save(customer);
+    }
+    tx.Commit();
+}]]></programlisting>
 
     <para>
-        This would fall over with an <literal>OutOfMemoryException</literal> somewhere 
```

Comment:
Ok, so

    git config --global filter.trimWhitespace.clean git-stripspace

Will not be a good idea? ^^